odoo/api.py

```diff
@@ -75,8 +75,8 @@ def __new__(meta, name, bases, attrs):
                 # make the method inherit from decorators
                 value = propagate(getattr(parent, key, None), value)
 
-                if (getattr(value, '_api', None) or '').startswith('cr'):
-                    _logger.warning("Deprecated method %s.%s in module %s", name, key, attrs.get('__module__'))
+                if key == '_execute_sql':
+                    _logger.warning("Deprecated method %s._execute_sql in module %s", name, attrs.get('__module__'))
```

Comment:
This metaclass only deals with API conversion, and does not care about actual method names.
I suggest to move this check into `odoo.models.MetaModel.__init__`.

Reply:
Yep, thanks, done. I still remove the old depreciated warning in the super class of MetaModel (Meta in api.py). It is ok for you ?